docs/design/scheduler-configuration.md

```diff
@@ -110,16 +110,22 @@ The first rule that matches, i.e. returns a fully qualified queue name, will hal
 Rules provide a fully qualified queue name as the result. To allow for deeper nesting of queues the parent of the queue can be set as part of the rule evaluation. The rule definition should allow a fixed configured fully qualified parent to be specified or it can call a second rule to generate the parent queue.  By default a queue is generated as a child of the root queue.
 
 Example:
+Placing an application submitted by the user _user1_ whom is a member of the groups _user1_ and _companyA_ in a queue based on UserName:
 ```
-Placing an application submitted by the user user1 whom is a member of the groups companyA and test in a queue based on UserName
 Rule name: UserName
-Parent: root.fixedparent
+    Parent: root.fixedparent
 Result: root.fixedparent.user1
 
 Rule name: UserName
-Parent: SecondaryGroup
-	Filter: company*
+    Parent: SecondaryGroup
+	Filter: 
+	    Groups: company.*
 Result: root.companyA.user1
+
+Rule name: UserName
+Filter: 
+    Users: user2,user3
+Result: denied placement
 ```
```

Comment:
In addition to this, we need to add more examples for other queue mapping policies, like create queues, namespace, etc. We can add documentations in a separate JIRA.

Reply:
yes, already in progress #31